optuna/notebooks/optuna_rapids.ipynb

```diff
@@ -80,7 +82,7 @@
    "source": [
     "## Set up CUDA Cluster\n",
     "\n",
-    "We start a local cluster and keep it ready for running distributed tasks with dask.\n",
+    "We start a local cluster and keep it ready for running distributed tasks with dask. The dask scheduler can help leverage multiple nodes available on the cluster.\n",
```

Comment:
```suggestion
    "We start a local cluster and keep it ready for running multiple workers on the current node.\n",
```
The current location of the multinode thing seems a little confusingly close to the LocalCudaCluster usage.

Then maybe change "Learn more: " in the next paragraph to say something like "Dask can also be used to run a cluster across multiple nodes, leveraging Kubernetes, YARN, or just a manual cluster setup. You can learn more : " (include link)